.idea/misc.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectType">
+    <option name="id" value="jpab" />
+  </component>
+  <component name="SwUserDefinedSpecifications">
+    <option name="specTypeByUrl">
+      <map />
+    </option>
+  </component>
+</project>
```

Comment:
I think this file is a bit weird. I think `jpab` refers to some java stuff.

Was this on purpose? My CLion does not add this file.

Reply:
I am sorry this was a mistake, is it ok if I make a new commit and then we squash this pull request before merging? Same goes for the `project.pbxproj` file. I have inadvertently commited these two files...

Reply:
Sure, not problem! I think squashing is the preferred merge option with small things.